nsxt/data_source_nsxt_policy_transport_zone.go

```diff
@@ -41,11 +43,49 @@ func dataSourceNsxtPolicyTransportZone() *schema.Resource {
 				Computed:     true,
 				ValidateFunc: validation.StringInSlice(policyTransportZoneTransportTypes, false),
 			},
+			"site_path": {
+				Type:         schema.TypeString,
+				Description:  "Path of the site this Transport Zone belongs to",
+				Optional:     true,
+				ValidateFunc: validatePolicyPath(),
+			},
 		},
 	}
 }
 
 func dataSourceNsxtPolicyTransportZoneRead(d *schema.ResourceData, m interface{}) error {
+	objSitePath := d.Get("site_path").(string)
+	transportType := d.Get("transport_type").(string)
+	if !isPolicyGlobalManager(m) && objSitePath != "" {
+		return globalManagerOnlyError()
+	}
+	if isPolicyGlobalManager(m) {
+		if objSitePath == "" {
+			return attributeRequiredGlobalManagerError()
```

Comment:
We should specify which attribute is required

Reply:
Sure, fixing. Thanks!